substrate/primitives/genesis-builder/src/lib.rs

```diff
@@ -66,6 +66,10 @@ pub type PresetId = sp_runtime::RuntimeString;
 /// [`GenesisBuilder`] interface.
 pub const DEV_RUNTIME_PRESET: &'static str = "development";
 
+/// The default `local-testnet` preset used to communicate with the runtime via
+/// [`GenesisBuilder`] interface.
+pub const LOCAL_TESTNET_RUNTIME_PRESET: &'static str = "local_testnet";
```

Comment:
Sorry for nit-picking: maybe it would good to add some note on which preset is mandatory and which is optional?

Seems that we kinda require "development" (e.g. for benchmarking, perhaps for omni-node - not sure what is the status there?).

And "local_testnet" is just for "our" runtimes? Not really required anywhere in generic tools?

Reply:
I see those constants as a unified good common "examples". It is hard to say which is mandatory or optional here.
I mean, the module `sp-genesis-builder` does not even know that some `frame-omni-bencher` exists and uses those constants.